frontend/src/components/shared/Footer.js

```diff
@@ -1,7 +1,98 @@
 import React from 'react'
+import { Box, Typography, Link } from '@mui/material'
+import { AiOutlineMail, AiOutlineLinkedin, AiOutlineInstagram, AiFillGithub } from 'react-icons/ai'
+import Tooltip from '@mui/material/Tooltip';
 
 function Footer() {
-    return <div>-footer component placeholder-</div>
+    return (
+        <main>
+            <div>
+                <Box sx={{ display: "flex", flexDirection: "row", bgcolor: "secondary.dark", p: 3 }}>
+                    <Box component="img"
+                        sx={{ height: 50, width: 45 }}
+                        src="/assets/bp-logo.png" />
+                    <Typography fontSize={20} color='primary.light' mt={0.5} ml={0.5}>blueprint</Typography>
+                    <Box sx={{
+                        display: "flex",
+                        flexDirection: "row",
+                        flexGrow: 1,
+                        flexWrap: 'wrap',
+                        justifyContent: "flex-end",
+                    }}>
+                        <Box sx={{ mx: 5, justifyContent: "center" }}>
+                            <Typography variant='h6' color='primary.light' mb={2} mr={1} fontWeight='medium'> General </Typography>
```

Comment:
can we make the header (General, For Students, For Nonprofits, Contact us) same size as the the other text but in bold